libs/packages/sam-formly/src/lib/formly/wrappers/form-field.wrapper.ts

```diff
@@ -32,9 +32,9 @@ import { FieldWrapper } from '@ngx-formly/core';
         <span>{{ to.label }}</span>
         <span *ngIf="!to.required && !to.hideOptional"> (Optional)</span>
       </label>
-      <small *ngIf="to.description" class="form-text text-muted">{{
-        to.description
-      }}</small>
+      <div *ngIf="to.description" class="usa-hint">
+        {{ to.description }}
+      </div>
```

Comment:
Two questions on this:
Why use two separate classes for description? - `form-text text-muted` and `usa-hint`
Will the the usa-label--description class be in sam-styles? I know that class shouldn't be on ngx-uswds, but I assumed it was created for sds.

Reply:
Udated the class to usa-label--description